src/Simplex/Chat.hs

```diff
@@ -234,10 +279,12 @@ processAgentMessage User {userId, profile} agentConnId agentMessage = do
         setActive $ ActiveC c
       _ -> pure ()
 
-    groupInvitation :: Contact -> (MemberId, GroupMemberRole) -> (MemberId, GroupMemberRole) -> SMPQueueInfo -> GroupProfile -> m ()
-    groupInvitation _ct (fromMemId, fromRole) (memId, memRole) _qInfo _groupProfile = do
+    saveGroupInvitation :: Contact -> GroupInvitation -> m ()
+    saveGroupInvitation ct@Contact {localDisplayName} inv@(GroupInvitation (fromMemId, fromRole) (memId, memRole) _ _) = do
       when (fromRole < GRAdmin || fromRole < memRole) $ throwError $ ChatError CEGroupRole
```

Comment:
and here it might be strange to receive such error as its name implies that it user's role that is insufficient.. maybe have a separate error - e.g. "Non Admin Invitation"

Reply:
the description `"insufficient role for this group command"` also clearly implies it's user's permissions that are lacking

Reply:
CEGroupHostRole ?